.github/workflows/warm_ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Test WARM
+
+on: push
+
+jobs:
+  build:
+    name: build
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: installing llvm
+      run: sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+    - name: creating symlinks
+      run: |
+        sudo ln -svf /usr/bin/clang-9 /usr/bin/clang
+        sudo ln -svf /usr/bin/wasm-ld-9 /usr/bin/wasm-ld
+        sudo ln -svf /usr/bin/llc-9 /usr/bin/llc
+        sudo ln -svf /usr/bin/llvm-nm-9 /usr/bin/llvm-nm
+        sudo ln -svf /usr/bin/llvm-ar-9 /usr/bin/llvm-ar
+    - name: building wasi-libc
+      run: |
+        git clone https://github.com/WebAssembly/wasi-libc.git
+        cd wasi-libc
+        make WASM_CC=/usr/bin/clang WASM_AR=/usr/bin/llvm-ar WASM_NM=/usr/bin/llvm-nm
+    - name: running script
+      run: |
+        cd /home/runner/work/warm/warm/
```

Comment:
I don't think, this is necessary, according to the documentation at https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun :

> Each run keyword represents a new process and shell in the runner environment. 

Thus, I'd assume, that all `run`s start from the same default working directory.